flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
because to build appimages I use appimagetool, and in the previous versions i didn't think of a way to use both appimagetoolx86_64 and arm, but I'm adding it now